.cirrus.yml

```diff
@@ -190,7 +153,21 @@ build_task:
         folder: *gopath  # Required hard-coded path, no variables.
         fingerprint_script: echo "$_BUILD_CACHE_HANDLE"
         # Cheat: Clone here when cache is empty, guaranteeing consistency.
-        populate_script: *full_clone
+        populate_script: &full_clone |
+          cd /
+          rm -rf $CIRRUS_WORKING_DIR
+          mkdir -p $CIRRUS_WORKING_DIR
+          git clone --recursive --branch=$DEST_BRANCH https://x-access-token:${CIRRUS_REPO_CLONE_TOKEN}@github.com/${CIRRUS_REPO_FULL_NAME}.git $CIRRUS_WORKING_DIR
+          cd $CIRRUS_WORKING_DIR
+          git remote update origin
+          if [[ -n "$CIRRUS_PR" ]]; then # running for a PR
+              git fetch origin pull/$CIRRUS_PR/head:pull/$CIRRUS_PR
+              git checkout pull/$CIRRUS_PR
+          else
+              git reset --hard $CIRRUS_CHANGE_IN_REPO
+          fi
+          cd $CIRRUS_WORKING_DIR
```

Comment:
I realize this is copy-paste, and the error is in the original, but please remove this unnecessary duplicate `cd`

Reply:
done